api_blog/app.py

```diff
@@ -24,4 +26,13 @@ async def create_app(config):
     aioreloader.start()
     app["config"] = config
     setup_routes(app)
+    app.cleanup_ctx.append(pg_context)
     return app
+
+
+async def pg_context(app):
+    url = app["config"]["db_url"]
```

Comment:
I don't see this var in config.yaml, it's only in your local config

Reply:
Yes, this var in local.yaml, it should be different?